src/style.css

```diff
@@ -1,3 +1,68 @@
-body {
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
+
+html, body {
+  font-family: 'Roboto', sans-serif;
```

Comment:
Consider adding a fallback font to ensure that text is displayed correctly if 'Roboto' is unavailable. For example: `font-family: 'Roboto', Arial, sans-serif;`